packages/plexus/package.json

```diff
@@ -57,9 +57,9 @@
     "build":
       "NODE_ENV=production npm-run-all -ln --serial _tasks/clean/* _tasks/bundle-worker --parallel _tasks/build/**",
     "coverage": "echo 'NO TESTS YET'",
-    "prepublishOnly": "$npm_execpath build",
+    "prepublishOnly": "yarn build",
     "start":
-      "NODE_ENV='development' npm-run-all -ln --serial _tasks/clean/worker --parallel '_tasks/bundle-worker --watch' _tasks/dev-server",
+      "NODE_ENV='development' npm-run-all -ln --serial _tasks/clean/worker _tasks/bundle-worker --parallel '_tasks/bundle-worker --watch' _tasks/dev-server",
```

Comment:
Why does `_tasks/bundle-worker` need to be run before the parallel that includes it with the `--watch` flag? after a quick google it seems like running webpack with the `--watch` flag should include an initial build.

Reply:
Because the bundle-worker task finishes after the dev-server, so the file is not present for the dev-server task, initially. This causes the initial build of dev-server to fail. And, when the bundle-worker finishes, the change is not detected by dev-server.